package.json

```diff
@@ -72,6 +72,7 @@
     ]
   },
   "devDependencies": {
+    "coffeescript": "^2.6.1",
```

Comment:
Now this is a name I haven't heard in a while.
How/where are we using coffeescript?